notebooks/failure-type-classification/failure_type_functions.ipynb

```diff
@@ -0,0 +1,328 @@
+{
```

Comment:
<p>Can you update this markdown to reflect this notebook is just the helper functions? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/284/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>